rubocop-sequel.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.5'
 
-  gem.add_runtime_dependency 'rubocop', '~> 1.0'
+  gem.add_dependency 'rubocop', '~> 1.0'
```

Comment:
Had to make this change due to [failed CI build](https://github.com/rubocop/rubocop-sequel/actions/runs/11420687878/job/31781231537)

![image](https://github.com/user-attachments/assets/4dc834d8-df1e-43a9-8237-56c24f1d2395)